app/assets/javascripts/todo_lists/todo_list_item_form.js

```diff
@@ -0,0 +1,62 @@
+$(document).ready(function() {
+
+  $('#todo-list-item-form').on('submit', function(event) {
```

Comment:
function can be declared separately like this:
```
var handleSubmit = function(event) { };
$('#todo-list-item-form').on('submit', handleSubmit);
```

to avoid nesting